src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -229,4 +229,32 @@ public function testHostnameIsProperlyParsed()
 
         $this->assertNoViolation();
     }
+
+    public function provideCheckTypes()
```

Comment:
Can you move the provider after the test (that's the convention we are using in Symfony).